pkg/ciliumenvoyconfig/cec_resource_parser.go

```diff
@@ -526,6 +526,7 @@ func (r *cecResourceParser) getBPFMetadataListenerFilter(useOriginalSourceAddr b
 		BpfRoot:                  bpf.BPFFSRoot(),
 		IsL7Lb:                   l7lb,
 		ProxyId:                  uint32(proxyPort),
+		PolicyUpdateWarningLimit: durationpb.New(option.Config.FQDNProxyResponseMaxDelay),
```

Comment:
import `durationpb` is missing

Reply:
Had it left in the work tree... updated.